ckanext/unaids/react/components/BulkFileUploaderComponent/src/App.js

```diff
@@ -0,0 +1,201 @@
+import React, { useState, useEffect } from 'react';
+import { Promise } from 'bluebird';
+import axios from 'axios';
+import FileUploader from './FileUploader';
+import { Client } from 'giftless-client';
+import ProgressBar from './ProgressBar';
+
+export default function App({ lfsServer, maxResourceSize, orgId, datasetId, defaultFields }) {
```

Comment:
Try to organize your react components into separate files for better readability.